circle.yml

```diff
@@ -14,10 +14,12 @@ jobs:
     steps:
       - checkout
       - run: bundle install
+      - run: which rsync || sudo apt-get install rsync
```

Comment:
rsync è portato sù poichè utilizzato in `./_buildscripts/build-swagger.sh` (che deve essere avviato prima di `jekyll build`)